tests/tck/features/lookup/ByIndex.feature

```diff
@@ -585,23 +585,20 @@ Feature: Lookup by index itself
       | name          |
       | "Tim Duncan"  |
       | "Tony Parker" |
+    # (TODO) Unsupported cases due to the lack of float precision
+    When executing query:
+      """
+      LOOKUP ON weight WHERE weight.WEIGHT > 70.5 YIELD id(vertex) as name;
```

Comment:
YIELD `weight.WEIGHT` for more intuitive validation. Any issue related to float precision here?